tailcfg/tailcfg.go

```diff
@@ -1330,9 +1329,9 @@ type MapResponse struct {
 	// ControlTime, if non-zero, is the current timestamp according to the control server.
 	ControlTime *time.Time `json:",omitempty"`
 
-	// TKA, if non-nil, describes updates for the local instance of the
-	// tailnet key authority.
-	TKA *TKAMapResponse `json:",omitempty"`
+	// TKAInstructions, if non-nil, describes updates for the local instance
+	// of the tailnet key authority.
+	TKA *TKAInstructions `json:",omitempty"`
```

Comment:
We'd discussed using the c2n http mechanism from control to node for this, no? Or is this something else?

Reply:
Is everything in map request/response going to be moving to c2n? We should use whichever mechanism is going to be appropriate for tailnet-wide 'settings' changes and the like.

Reply:
depends on whether it's imperative or declarative.

declarative stuff will stay on MapResponse.

But "do this" will be c2n.

Reply:
WRT our offsite discussion: I was mostly thinking of using c2n for doing things like asking a node to direct-sign a rotation-signature, which fits a bit better into the c2n model.

Reply:
Okay gotcha, then the 'plz resign' stuff will be c2n, and TKAInstructions should stay in <notification-channel> / map response